### PR TITLE
Ambiclimate

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22827

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ambiclimate.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: "Ambiclimate A/C controller"
+description: "Instructions on how to integrate Ambiclimate A/C controller into Home Assistant."
+date: 2019-03-21 15:00 +0200
```
Comment (klaasnicolaas) [inline source/_components/ambiclimate.markdown]: Please do not change the date.

Comment (Danielhiversen) [reply source/_components/ambiclimate.markdown]: ok, why?

Comment (klaasnicolaas) [reply source/_components/ambiclimate.markdown]: it's only for when the page is created, not for when the component will be released 😉

Comment (Danielhiversen) [reply source/_components/ambiclimate.markdown]: Ok, but I do not understand why it matters as long as it is a new file.

**Comment by frenck** [comment]
closing this PR, since the parent PR has been closed.

**Comment by Danielhiversen** [comment]
New pr https://github.com/home-assistant/home-assistant/pull/22827

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**